### ISSUE TITLE
Potentially related issues

### ISSUE THREAD

**Issue by LivesParkkk** [GingerWalnut/SQ5.0Public#101]
This has happened twice so far. In my colony on Kattadi, I will be able to kill mobs etc with a bow perfectly fine. Then, however, my arrows just stop being able to affect mobs in the colony. I have not tested outside the colony. The arrows simply bounce off the mob. Then, at the same time, all my drills simultaneously stop but continue consuming power; their charge goes down but they do not move or break blocks, even though there is no hole, no water, or anything to stop their progress. Last night Ginger updated the server and everything was fixed post restart, until now, where the issue has reappeared,

TLDR: Arrows stop hitting mobs drills stop mining but take power, fixed by restart potentially?

**Comment by LivesParkkk** [comment]
Rufiani is also experiencing this issue.

**Comment by Laserlance141** [comment]
This might be a duplicate of #9 as invulnerable mobs have arrows bounce off them.

**Comment by LivesParkkk** [comment]
#9 also occurs on Katadi with spiders.

**Comment by GingerWalnut** [comment]
Seems to have been fixed, please let me know if it hasn't

**Comment by LivesParkkk** [comment]
Still Happening on Kattadi right now

**Comment by GingerWalnut** [comment]
Should be fixed for real this time
